Model/Logger.php

```diff
@@ -77,24 +110,33 @@ public function getPath()
      * @throws LocalizedException
      * @return void
      */
-    public function log($message, $label = '') {
-        try {
-            if (!empty($label)) {
-                $label = '[' . strtoupper($label) . '] ';
-            }
-            $timestamp = date('d M Y H:i:s', time());
-            $message = sprintf('%s%s - %s%s', PHP_EOL, $timestamp, $label, $message);
-            $this->driverFile->filePutContents($this->getPath(), $message, FILE_APPEND);
-            if ($this->isRecording) {
-                $this->playback[] = $message;
-            }
-            if ($this->console) {
-                $this->console->write($message);
+    public function log($message, $label = '')
+    {
+        if ($this->scopeConfig->getValue(TaxjarConfig::TAXJAR_DEBUG, \Magento\Store\Model\ScopeInterface::SCOPE_STORE, $this->storeManager->getStore()->getId())) {
```

Comment:
This line seems a bit long. Should we assign a variable and / or break those args on to multiple lines?